{"name": "Проверка создания уникального пользователя", "status": "failed", "statusDetails": {"message": "AssertionError: assert 403 == 200", "trace": "self = <tests.test_create_user.TestCreateUser object at 0x000001822215F610>\ngenerate_user_data = [{'message': 'User already exists', 'success': False}, 403, 'qsanchez@example.org', '%C#j_3Plrh', 'David Sweeney']\n\n    @allure.title(\"Проверка создания уникального пользователя\")\n    def test_create_unique_user(self, generate_user_data):\n        body = generate_user_data[0]\n        status_code = generate_user_data[1]\n>       assert status_code == 200\nE       assert 403 == 200\n\ntests\\test_create_user.py:12: AssertionError"}, "start": 1747143200379, "stop": 1747143200379, "uuid": "437390bb-f896-4c65-96d1-dc47362b2bbc", "historyId": "d019e29863ce32a43478f9f92dd0d2ef", "testCaseId": "d019e29863ce32a43478f9f92dd0d2ef", "fullName": "tests.test_create_user.TestCreateUser#test_create_unique_user", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_create_user"}, {"name": "subSuite", "value": "TestCreateUser"}, {"name": "host", "value": "DESKTOP-B0AK6U5"}, {"name": "thread", "value": "4380-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_create_user"}]}